<!doctype html><!-- This site was created with Wowchemy. https://www.wowchemy.com --><!-- Last Published: January 16, 2026 --><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=generator content="Wowchemy 5.7.0 for Hugo"><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload='this.media="all"'><link rel=stylesheet href=/css/vendor-bundle.min.047268c6dd09ad74ba54a0ba71837064.css media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/academicons@1.9.2/css/academicons.min.css integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=/css/wowchemy.042e26407c9e383d96a1f26d6787c686.css><link rel=stylesheet href=/css/libs/chroma/github-light.min.css title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=/css/libs/chroma/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled><meta name=author content="Laurent Jacob"><meta name=description content="A researcher on machine learning and statistics for genomics"><link rel=alternate hreflang=en-us href=https://laurent-jacob.github.io/><link rel=canonical href=https://laurent-jacob.github.io/><link rel=manifest href=/manifest.webmanifest><link rel=icon type=image/png href=/media/icon_hu0c3dae586d456983f2f0bb7fb2bda700_1591_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu0c3dae586d456983f2f0bb7fb2bda700_1591_180x180_fill_lanczos_center_3.png><meta name=theme-color content="#1565c0"><meta property="twitter:card" content="summary"><meta property="twitter:site" content="@ljacob"><meta property="twitter:creator" content="@ljacob"><meta property="twitter:image" content="https://laurent-jacob.github.io/media/icon_hu0c3dae586d456983f2f0bb7fb2bda700_1591_512x512_fill_lanczos_center_3.png"><meta property="og:site_name" content="Laurent Jacob"><meta property="og:url" content="https://laurent-jacob.github.io/"><meta property="og:title" content="Laurent Jacob"><meta property="og:description" content="A researcher on machine learning and statistics for genomics"><meta property="og:image" content="https://laurent-jacob.github.io/media/icon_hu0c3dae586d456983f2f0bb7fb2bda700_1591_512x512_fill_lanczos_center_3.png"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2023-09-11T00:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"WebSite","potentialAction":{"@type":"SearchAction","target":"https://laurent-jacob.github.io/?q={search_term_string}","query-input":"required name=search_term_string"},"url":"https://laurent-jacob.github.io/"}</script><script src=https://identity.netlify.com/v1/netlify-identity-widget.js></script><link rel=alternate href=/index.xml type=application/rss+xml title="Laurent Jacob"><title>Laurent Jacob</title></head><body id=top data-spy=scroll data-offset=70 data-target=#navbar-main class=page-wrapper data-wc-page-id=3976528693a0108357f4928017600865><script src=/js/wowchemy-init.min.ec9d49ca50e4b80bdb08f0417a28ed84.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class="page-header header--fixed"><header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Laurent Jacob</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Laurent Jacob</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about data-target=#about><span>About</span></a></li><li class=nav-item><a class=nav-link href=/#projects data-target=#projects><span>Projects</span></a></li><li class=nav-item><a class=nav-link href=/#people data-target=#people><span>People</span></a></li><li class=nav-item><a class=nav-link href=/#publications data-target=#publications><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#community data-target=#community><span>Community</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span>
</a><a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span>
</a><a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></header></div><div class=page-body><span class="js-widget-page d-none"></span><section id=about class="home-section wg-about-biography"><div class=home-section-bg></div><div class=container><div class=row><div class="col-12 col-lg-4"><div id=profile><img class="avatar avatar-circle" width=270 height=270 src=/authors/admin/avatar_hu63c7b36fbab3c1a672794e82fc633921_1358325_270x270_fill_lanczos_center_3.png alt="Laurent Jacob"><div class=portrait-title><h2>Laurent Jacob</h2><h3>Researcher in machine learning and statistics for genomics</h3></div><ul class=network-icon aria-hidden=true><li><a href="https://scholar.google.com/citations?hl=fr&amp;user=54bOCmQAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" target=_blank rel=noopener aria-label=google-scholar><i class="ai ai-google-scholar big-icon"></i></a></li></ul></div></div><div class="col-12 col-lg-8"><div class=article-style><p>I am a senior <a href=https://www.cnrs.fr/en target=_blank rel=noopener>CNRS</a> researcher located at
the <a href=http://www.lcqb.upmc.fr/ target=_blank rel=noopener>LCQB</a> lab of <a href=https://www.sorbonne-universite.fr/en target=_blank rel=noopener>Sorbonne
Université</a> in Paris.</p><p>I work on machine learning and statistics to answer questions in
genomics. I am particularly interested in microbial genomics, notably
to better understand and predict drug resistances, and in evolutionary
genomics. Part of my work also aim at better understanding neural
networks for biological sequences. See the <a href=/#projects>Projects</a>
section for some examples.</p><p>I received an MSc in machine learning from ENS Cachan (now ENS
Paris-Saclay) in 2006, and a Ph.D. from Mines ParisTech in 2009, where
I worked on regularized machine learning for computational
biology. Between 2010 and 2012 I was a postdoctoral fellow at the
statistics department of UC Berkeley. I joined the CNRS in 2013 and
worked at the <a href=https://lbbe.univ-lyon1.fr/en target=_blank rel=noopener>LBBE</a> lab in <a href=https://www.univ-lyon1.fr/en target=_blank rel=noopener>Université
Lyon 1</a> until 2023.</p><p>You can reach me at firstname (dot) lastname (at) cnrs (dot) fr.</p></div><div class=row></div></div></div></div></section><section id=projects class="home-section wg-portfolio"><div class=home-section-bg></div><div class=container><div class="row justify-content-center"><div class="section-heading col-12 mb-3 text-center"><h1 class=mb-0>Projects</h1></div><div class=col-12><span class="d-none default-project-filter">*</span><div class=project-toolbar><div class=project-filters><div class="btn-toolbar d-flex justify-content-center"><div class="btn-group flex-wrap"><a href=# data-filter=* class="btn btn-primary btn-lg active">All</a>
<a href=# data-filter=.js-id-deep-learning class="btn btn-primary btn-lg">Deep Learning</a>
<a href=# data-filter=.js-id-gwas class="btn btn-primary btn-lg">GWAS</a></div></div></div></div><div class="isotope projects-container row js-layout-row project-showcase"><div class="col-12 isotope-item js-id-deep-learning"><div class="col-lg-12 mb-5 view-showcase"><div class="row align-items-center"><div class="col-12 col-md-6"><div class="section-subheading article-title mb-0 mt-0">Phyloformer</div><div class=article-style>W﻿e train neural networks on sets of homologous sequences simulated from probabilistic evolution models to infer phylogenetic trees. We use self-attention to obtain permutation-equivariant functions and ensure that the obtained tree does not depend on the order in which the sequences are provided. Our hope is to perform inference that is order of magnitude faster than state of the art maximum likelihood, and as accurate or even better under models with untractable likelihoods.</div><div class=btn-links><a class="btn btn-outline-primary btn-page-header" href=https://www.biorxiv.org/content/10.1101/2022.06.24.496975v1 target=_blank rel=noopener>PDF
</a><a class="btn btn-outline-primary btn-page-header" href=https://github.com/lucanest/Phyloformer target=_blank rel=noopener>Code
</a><a class="btn btn-outline-primary btn-page-header" href="https://www.youtube.com/watch?v=ytqRFZDg1vI" target=_blank rel=noopener>Video</a></div></div><div class="col-12 col-md-6 order-first"><img src=/project/phyloformer/featured_hu4d8c52e035119771581449c353f4bf4b_95587_540x0_resize_q75_h2_lanczos_3.webp height=258 width=540 alt=Phyloformer loading=lazy></div></div></div></div><div class="col-12 isotope-item js-id-gwas js-id-deep-learning"><div class="col-lg-12 mb-5 view-showcase"><div class="row align-items-center"><div class="col-12 col-md-6"><div class="section-subheading article-title mb-0 mt-0">SEISM</div><div class=article-style>Convolutional neural networks on biological sequences are known to learn probabilistic motifs associated with the target phenotype. To go beyond this informal interpretation, we provide a statistical test quantifying the motif-phenotype association. This requires to solve a post-selection inference problem, where the selection involves the infinite set of possible motifs.</div><div class=btn-links><a class="btn btn-outline-primary btn-page-header" href="https://openreview.net/pdf?id=nddEHTSnqg" target=_blank rel=noopener>PDF
</a><a class="btn btn-outline-primary btn-page-header" href=https://gitlab.in2p3.fr/antoine.villie1/seism target=_blank rel=noopener>Code</a></div></div><div class="col-12 col-md-6 order-first order-md-2"><img src=/project/seism/featured_hu447cc4a7f8dbb876af50fc6570d123ab_23870_540x0_resize_q75_h2_lanczos_3.webp height=163 width=540 alt=SEISM loading=lazy></div></div></div></div><div class="col-12 isotope-item js-id-gwas"><div class="col-lg-12 mb-5 view-showcase"><div class="row align-items-center"><div class="col-12 col-md-6"><div class="section-subheading article-title mb-0 mt-0">DBGWAS</div><div class=article-style>W﻿e describe the variation in bacterial genomes through their content in short sequences (k-mers). This avoids the bias usually caused by focusing on pre-defined SNPs or gene lists. We use this representation to pinpoint the genomic variation associated to antimicrobial resitances, and rely on a so-called de Bruijn graph to help make sense of the identified k-mers in terms of SNPs or mobile genetic elements.</div><div class=btn-links><a class="btn btn-outline-primary btn-page-header" href=https://pubmed.ncbi.nlm.nih.gov/30419019/ target=_blank rel=noopener>PDF
</a><a class="btn btn-outline-primary btn-page-header" href=https://gitlab.com/leoisl/dbgwas target=_blank rel=noopener>Code</a></div></div><div class="col-12 col-md-6 order-first"><img src=/project/dbgwas/featured_huc86fd43e13884e7e23d9798638eeb466_193344_540x0_resize_q75_h2_lanczos_3.webp height=283 width=540 alt=DBGWAS loading=lazy></div></div></div></div><div class="col-12 isotope-item js-id-gwas"><div class="col-lg-12 mb-5 view-showcase"><div class="row align-items-center"><div class="col-12 col-md-6"><div class="section-subheading article-title mb-0 mt-0">CALDERA</div><div class=article-style>A follow-up on DBGWAS. We use the structure of the de Bruijn graph to define new genomic variants (e.g. gene presence) that accomodate polymorphisms. This amounts to defining one variant for each connected subgraph. We rely on a reverse search strategy to efficiently enumerate variants, and on the Tarone strategy to control the FWER when testing their association with a bacterial phenotype.</div><div class=btn-links><a class="btn btn-outline-primary btn-page-header" href=https://academic.oup.com/bioinformatics/article/38/Supplement_1/i36/6617517 target=_blank rel=noopener>PDF
</a><a class="btn btn-outline-primary btn-page-header" href=https://hectorrdb.github.io/project/caldera/ target=_blank rel=noopener>Code
</a><a class="btn btn-outline-primary btn-page-header" href="https://www.youtube.com/watch?v=0AY8KBVDkCQ" target=_blank rel=noopener>Video</a></div></div><div class="col-12 col-md-6 order-first order-md-2"><img src=/project/caldera/featured_hue1185475340ece6d8df90279605c51c0_504316_540x0_resize_q75_h2_lanczos_3.webp height=460 width=540 alt=CALDERA loading=lazy></div></div></div></div><div class="col-12 isotope-item js-id-deep-learning"><div class="col-lg-12 mb-5 view-showcase"><div class="row align-items-center"><div class="col-12 col-md-6"><div class="section-subheading article-title mb-0 mt-0">Convolutional kernel networks</div><div class=article-style>W﻿e analyze convolutional and recurrent neural networks for biological sequences in the framework of positive definite kernels. In addition to providing a new interpretation on the genomic features underlying these networks, this analysis provides a version that is better regularized and performs better when few data is available. We also extended this work to graph-structured data.</div><div class=btn-links><a class="btn btn-outline-primary btn-page-header" href=https://academic.oup.com/bioinformatics/article/35/18/3294/5308597 target=_blank rel=noopener>PDF
</a><a class="btn btn-outline-primary btn-page-header" href=https://gitlab.inria.fr/dchen/CKN-seq target=_blank rel=noopener>Code</a></div></div><div class="col-12 col-md-6 order-first"><img src=/project/ckn/featured_hu481d538f82141abe0cbb0696643a802e_85112_540x0_resize_q75_h2_lanczos_3.webp height=303 width=540 alt="Convolutional kernel networks" loading=lazy></div></div></div></div></div></div></div></div></section><section id=people class="home-section wg-people"><div class=home-section-bg></div><div class=container><div class="row justify-content-left people-widget"><div class="section-heading col-12 mb-3 text-center"><h1 class=mb-0>People</h1></div><div class=col-md-12>I am grateful to all the people I co-advised. Most of the work featured on this website is theirs.</div><div class="col-md-12 text-left"><h2 class=mb-4>Ph.D. students</h2></div><div class="col-12 col-sm-auto people-person"><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/amelie/avatar_hu63e7d68794864eab40cc7f1f13160f5e_97141_270x270_fill_q75_lanczos_center.jpeg alt=Avatar , style=margin-left:0;margin-right:auto><div class=portrait-title , style=text-align:left><h2>Amélie Leroy</h2><h3>2024-, with Hélène Morlon.</h3></div></div><div class="col-12 col-sm-auto people-person"><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/enzo/avatar_hu56e8c3538211e8a8fcfcb8d302ab404c_2771451_270x270_fill_lanczos_center_3.png alt=Avatar , style=margin-left:0;margin-right:auto><div class=portrait-title , style=text-align:left><h2>Enzo Marsot</h2><h3>2024-, with Damien de Vienne and Bastien Boussau.</h3></div></div><div class="col-12 col-sm-auto people-person"><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/quentin/avatar_hu177e74e1616f12a7762f204c96625077_53466_270x270_fill_q75_lanczos_center.jpg alt=Avatar , style=margin-left:0;margin-right:auto><div class=portrait-title , style=text-align:left><h2>Quentin Chaung</h2><h3>2024-, with Samuel Alizon.</h3></div></div><div class="col-12 col-sm-auto people-person"><a href=https://www.college-de-france.fr/fr/personne/vincent-garot target=_blank rel=noopener><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/vincent/avatar_hud8c827fa9ba95f2b0209d0eb8dacece6_181986_270x270_fill_lanczos_center_3.png alt=Avatar , style=margin-left:0;margin-right:auto></a><div class=portrait-title , style=text-align:left><h2><a href=https://www.college-de-france.fr/fr/personne/vincent-garot target=_blank rel=noopener>Vincent Garot</a></h2><h3>2023-, with Samuel Alizon and Anna Zhukova.</h3></div></div><div class="col-12 col-sm-auto people-person"><a href=https://lbbe.univ-lyon1.fr/en/node/78284 target=_blank rel=noopener><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/luca/avatar_hu7b8f53d107ccf88a8dc654a1e8cb515d_756511_270x270_fill_lanczos_center_3.png alt=Avatar , style=margin-left:0;margin-right:auto></a><div class=portrait-title , style=text-align:left><h2><a href=https://lbbe.univ-lyon1.fr/en/node/78284 target=_blank rel=noopener>Luca Nesterenko</a></h2><h3>2021-2024, with Bastien Boussau.</h3></div></div><div class="col-12 col-sm-auto people-person"><a href=https://fr.linkedin.com/in/antoine-villi%C3%A9 target=_blank rel=noopener><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/antoine/avatar_hu16125cd33a6381acd401f4634dbf3dff_68487_270x270_fill_lanczos_center_3.png alt=Avatar , style=margin-left:0;margin-right:auto></a><div class=portrait-title , style=text-align:left><h2><a href=https://fr.linkedin.com/in/antoine-villi%C3%A9 target=_blank rel=noopener>Antoine Villié</a></h2><h3>2019-2023, with Philippe Veber and Yohann de Castro.</h3><h3>Now a Data Science Manager at Aurobac Therapeutics</h3></div></div><div class="col-12 col-sm-auto people-person"><a href=https://dexiong.me/ target=_blank rel=noopener><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/dexiong/avatar_hud2e4c383ad0f66e133dd4a87739676f8_7696_270x270_fill_q75_lanczos_center.jpeg alt=Avatar , style=margin-left:0;margin-right:auto></a><div class=portrait-title , style=text-align:left><h2><a href=https://dexiong.me/ target=_blank rel=noopener>Dexiong Chen</a></h2><h3>2018-2020, with Julien Mairal.</h3><h3>Now a project leader at MPI Munich</h3></div></div><div class="col-12 col-sm-auto people-person"><a href=https://www.researchgate.net/profile/Magali-Jaillard target=_blank rel=noopener><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/magali/avatar_huddd7b065ffbe8c901fc819f2b49b5da9_80360_270x270_fill_q75_lanczos_center.jpg alt=Avatar , style=margin-left:0;margin-right:auto></a><div class=portrait-title , style=text-align:left><h2><a href=https://www.researchgate.net/profile/Magali-Jaillard target=_blank rel=noopener>Magali Jaillard</a></h2><h3>2016-2018, with Maud Tournoud and Pierre Mahé.</h3><h3>Now a research scientist at bioMérieux</h3></div></div><div class="col-md-12 text-left"><h2 class=mb-4>Postdocs and engineers</h2></div><div class="col-12 col-sm-auto people-person"><a href=https://lucblassel.com/ target=_blank rel=noopener><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/luc/avatar_hu25fc37cfad0485aa0ed74beff0d72f9b_737113_270x270_fill_lanczos_center_3.png alt=Avatar , style=margin-left:0;margin-right:auto></a><div class=portrait-title , style=text-align:left><h2><a href=https://lucblassel.com/ target=_blank rel=noopener>Luc Blassel</a></h2><h3>2023-, with Bastien Boussau.</h3></div></div><div class="col-12 col-sm-auto people-person"><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/johanna/avatar_hu7f1d9b2b1b7ae2f4f286bb2d32de8bca_111540_270x270_fill_q75_lanczos_center.jpg alt=Avatar , style=margin-left:0;margin-right:auto><div class=portrait-title , style=text-align:left><h2>Johanna Trost</h2><h3>2021-2023, with Bastien Boussau.</h3><h3>Now an MSc student</h3></div></div><div class="col-12 col-sm-auto people-person"><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/xavier/avatar_hu7cf27b4073176bc5df786f2b7270d060_1916_270x270_fill_lanczos_center_3.png alt=Avatar , style=margin-left:0;margin-right:auto><div class=portrait-title , style=text-align:left><h2>Xavier Castells Domingo</h2><h3>2015-2016, with Vincent Lacroix and Stéphane Robin.</h3></div></div><div class="col-md-12 text-left"><h2 class=mb-4>MSc interns</h2></div><div class="col-12 col-sm-auto people-person"><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/noemie/avatar_hu9c6eab5439e39a5cc9f943be9a666ab4_58850_270x270_fill_q75_lanczos_center.jpg alt=Avatar , style=margin-left:0;margin-right:auto><div class=portrait-title , style=text-align:left><h2>Noémie Sauvage</h2><h3>2025, with Luc Blassel.</h3></div></div><div class="col-12 col-sm-auto people-person"><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/louis-mael/avatar_hu7cf27b4073176bc5df786f2b7270d060_1916_270x270_fill_lanczos_center_3.png alt=Avatar , style=margin-left:0;margin-right:auto><div class=portrait-title , style=text-align:left><h2>Louis-Maël Guéguen</h2><h3>2022, with Camille Marchet.</h3><h3>Now a bioinformatician at CHUL in Québec</h3></div></div><div class="col-12 col-sm-auto people-person"><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/rosette/avatar_hu7cf27b4073176bc5df786f2b7270d060_1916_270x270_fill_lanczos_center_3.png alt=Avatar , style=margin-left:0;margin-right:auto><div class=portrait-title , style=text-align:left><h2>Rosette Von Raesfeldt</h2><h3>2014</h3></div></div><div class="col-12 col-sm-auto people-person"><a href=https://elsab.github.io/ target=_blank rel=noopener><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/elsa/avatar_hubcb457c87f4ccd30a72172a8b00bcb69_1595443_270x270_fill_q75_lanczos_center.jpg alt=Avatar , style=margin-left:0;margin-right:auto></a><div class=portrait-title , style=text-align:left><h2><a href=https://elsab.github.io/ target=_blank rel=noopener>Elsa Bernard</a></h2><h3>2012, with Julien Mairal and Jean-Philippe Vert.</h3><h3>Now a group leader at Institut Gustave Roussy</h3></div></div><div class="col-12 col-sm-auto people-person"><img width=270 height=270 loading=lazy class="avatar avatar-circle" src=/authors/anne-claire/avatar_hu8521199e7890c54eacdf987faed2161d_10192_270x270_fill_q75_lanczos_center.jpeg alt=Avatar , style=margin-left:0;margin-right:auto><div class=portrait-title , style=text-align:left><h2>Anne-Claire Haury</h2><h3>2009, with Jean-Philippe Vert.</h3><h3>Now a software engineer at Google research</h3></div></div></div></div></section><section id=publications class="home-section wg-collection"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Some recent publications</h1></div><div class="col-12 col-lg-8">See my <a href="https://scholar.google.com/citations?hl=fr&amp;user=54bOCmQAAAAJ" target=_blank rel=noopener>Google Scholar profile</a> for a complete list.<div class="view-list view-list-item"><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<a href=/publication/villie2023neural/>Neural Networks beyond explainability: Selective inference for sequence motifs</a><div class=article-metadata><span>Antoine Villié</span>, <span>Philippe Veber</span>, <span>Yohann de Castro</span>, <span class=author-highlighted>Laurent Jacob</span></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href="https://openreview.net/pdf?id=nddEHTSnqg" target=_blank rel=noopener>PDF
</a><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/villie2023neural/cite.bib>Cite
</a><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://gitlab.in2p3.fr/antoine.villie1/seism target=_blank rel=noopener>Code</a></div></div><div class="view-list view-list-item"><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<a href=/publication/roux2022caldera/>CALDERA: Finding all significant de Bruijn subgraphs for bacterial GWAS</a><div class=article-metadata><span>Hector Roux de Bézieux</span>, <span>Leandro Lima</span>, <span>Fanny Perraudeau</span>, <span>Arnaud Mary</span>, <span>Sandrine Dudoit</span>, <span class=author-highlighted>Laurent Jacob</span></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://academic.oup.com/bioinformatics/article/38/Supplement_1/i36/6617517 target=_blank rel=noopener>PDF
</a><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/roux2022caldera/cite.bib>Cite
</a><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://hectorrdb.github.io/project/caldera/ target=_blank rel=noopener>Code
</a><a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.youtube.com/watch?v=0AY8KBVDkCQ" target=_blank rel=noopener>Video</a></div></div><div class="view-list view-list-item"><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<a href=/publication/nesterenko2022phyloformer/>Phyloformer: towards fast and accurate phylogeny estimation with self-attention networks</a><div class=article-metadata><span>Luca Nesterenko</span>, <span>Bastien Boussau</span>, <span class=author-highlighted>Laurent Jacob</span></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.biorxiv.org/content/10.1101/2022.06.24.496975v1 target=_blank rel=noopener>PDF
</a><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/nesterenko2022phyloformer/cite.bib>Cite
</a><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/lucanest/Phyloformer target=_blank rel=noopener>Code
</a><a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.youtube.com/watch?v=ytqRFZDg1vI" target=_blank rel=noopener>Video</a></div></div><div class="view-list view-list-item"><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<a href=/publication/chen2019biological/>Biological sequence modeling with convolutional kernel networks</a><div class=article-metadata><span>Dexiong Chen</span>, <span>Julien Mairal</span>, <span class=author-highlighted>Laurent Jacob</span></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://academic.oup.com/bioinformatics/article/35/18/3294/5308597 target=_blank rel=noopener>PDF
</a><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/chen2019biological/cite.bib>Cite
</a><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://gitlab.inria.fr/dchen/CKN-seq target=_blank rel=noopener>Code</a></div></div><div class="view-list view-list-item"><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<a href=/publication/jaillard2018fast/>A fast and agnostic method for bacterial genome-wide association studies: Bridging the gap between k-mers and genetic events</a><div class=article-metadata><span>Magali Jaillard</span>, <span>Leandro Lima</span>, <span>Maud Tournoud</span>, <span>Pierre Mahé</span>, <span>Alex van Belkum</span>, <span>Vincent Lacroix</span>, <span class=author-highlighted>Laurent Jacob</span></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://pubmed.ncbi.nlm.nih.gov/30419019/ target=_blank rel=noopener>PDF
</a><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/jaillard2018fast/cite.bib>Cite
</a><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://gitlab.com/leoisl/dbgwas target=_blank rel=noopener>Code</a></div></div></div></div></div></section><section id=community class="home-section wg-markdown"><div class=home-section-bg></div><div class=container><div class="row justify-content-center"><div class="section-heading col-12 mb-3 text-center"><h1 class=mb-0>Community</h1></div><div class=col-12><p>I was recently involved in the organization of several scientific events:</p><ul><li><a href=https://legend2025.sciencesconf.org/ target=_blank rel=noopener>LEGEND 2025</a>, a conference on machine learning for evolutionary genomics.</li><li>I am co-chairing the Sequence Analysis track of the <a href=https://www.iscb.org/ismbeccb2025/home target=_blank rel=noopener>2025 ISMB conference</a>.</li><li><a href=https://legend2024.sciencesconf.org/ target=_blank rel=noopener>LEGEND 2024</a>, a conference on machine learning for evolutionary genomics.</li><li><a href=https://gt-lego.cnrs.fr/ target=_blank rel=noopener>LEGO</a>, a French working group on machine learning for genomics. We organize national meetings to gather researchers interested in this topic.</li><li><a href=https://mlmg2022.github.io/ target=_blank rel=noopener>MLMG 2022</a>, an <a href=https://2022.ecmlpkdd.org/index.html target=_blank rel=noopener>ECML</a> workshop on machine learning for microbial genomics.</li><li>I co-chaired the Systems Biology and Networks track of the <a href=https://www.iscb.org/ismbeccb2021 target=_blank rel=noopener>2021 ISMB conference</a>.</li><li>The 2020 <a href=https://databio.sciencesconf.org/ target=_blank rel=noopener>prospective meeting</a> on data science, AI and biology from the biology and computer science institutes of CNRS.</li></ul></div></div></div></section></div><div class=page-footer><div class=container><footer class=site-footer><p class=powered-by><a href=/terms/>Legal notice</a></p><p class="powered-by copyright-license-text">© 2026 Laurent Jacob. This work is licensed under <a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank>CC BY NC ND 4.0</a></p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank aria-label="Creative Commons"><i class="fab fa-creative-commons fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-by fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nc fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nd fa-2x" aria-hidden=true></i></a></p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-themes target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><script src=/js/vendor-bundle.min.938a3a7554cd9f6602290411f64d2617.js></script><script src=https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin=anonymous></script><script src=https://cdn.jsdelivr.net/gh/metafizzy/isotope@v3.0.6/dist/isotope.pkgd.min.js integrity="sha512-Zq2BOxyhvnRFXu0+WE6ojpZLOU2jdnqbrM1hmVdGzyeCa1DgM3X5Q4A/Is9xA1IkbUeDd7755dNNI/PzSf2Pew==" crossorigin=anonymous></script><script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script><script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script><script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script><script id=page-data type=application/json>{"use_headroom":false}</script><script src=/en/js/wowchemy.min.85070d5fe00d43eaedff44310b81dc2c.js></script><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy
</a><a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=/js/wowchemy-publication.af9327db0521d4a01354bfc8b77a4324.js type=module></script></body></html>